blocks/hooks/test/anchor.js

```diff
@@ -6,41 +6,31 @@ import { noop } from 'lodash';
 /**
  * WordPress dependencies
  */
-import { applyFilters, removeAllFilters } from '@wordpress/hooks';
```

Comment:
So the downside to this approach is that we can't clean up lingering handlers after tests?

Reply:
Test files are isolated, so they should be cleaned up anyways. I hope 😄

Reply:
>Test files are isolated, so they should be cleaned up anyways. I hope 😄

Hmm, the require cache is cleared between each? Would be (good) news to me if true.

Reply:
I think it isn’t the case only for mocks imported in Jest setup file.

You can enable reseting modules per test using [resetModules](https://facebook.github.io/jest/docs/en/configuration.html#resetmodules-boolean) config option or call it programmatically with the method called the same.